components/LanguageSelector/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react';
```

Comment:
I added this LanguageSelector component for the Header, so that we can make a request to the backend to update the user profile data once a user selects a different language